### ISSUE TITLE
Add PHPUnit 7.4 compatibility

### ISSUE THREAD

**Issue by willemstuursma** [php-mock/php-mock#21]
```
Fatal error: Declaration of phpmock\phpunit\MockObjectProxy::__phpunit_verify() must be compatible with PHPUnit_Framework_MockObject_MockObject::__phpunit_verify(bool $unsetInvocationMocker = true) in vendor/php-mock/php-mock-phpunit/classes/MockObjectProxy.php on line 17
```

**Comment by webimpress** [comment]
Please update php-mock-phpunit to version 2.1.2, released two days ago with PHPUnit 7.4 support.

**Comment by webimpress** [comment]
See: https://github.com/php-mock/php-mock-phpunit/pull/30/files#diff-f91b039a4359d4995852379724cd00d1R59

And release: 
https://github.com/php-mock/php-mock-phpunit/releases/tag/2.1.2
